app/server/appsmith-server/src/main/java/com/appsmith/server/migrations/ce/V002__loadMongoData.java

```diff
@@ -34,21 +34,45 @@ public class V002__loadMongoData extends AppsmithJavaMigration {
 
     private static final Pattern UUID_OR_OBJECTID_PATTERN =
             Pattern.compile("([\":])([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}|[0-9a-f]{24})(\")");
+    private static final String MONGO_DATA_NAME = "mongo-data";
 
     final Map<String, String> idMap = new HashMap<>();
 
-    final Path BASELINE_DATA_ROOT =
-            Path.of("/Users/shri/work/appsmith-ee-pg/deploy/docker/fs/opt/appsmith/utils/bin/baseline-"
-                    + ProjectProperties.EDITION.toLowerCase() + "-data");
+    private static Path findEffectiveDataPath() {
+        // MongoDB export placed in Stacks volume, in Docker container.
+        final Path mongoDataInStacks = Path.of("appsmith-stacks", MONGO_DATA_NAME);
```

Comment:
Looks like this is building relative paths. 🤦

```suggestion
        final Path mongoDataInStacks = Path.of("/", "appsmith-stacks", MONGO_DATA_NAME);
```